libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/RoadShield.kt

```diff
@@ -1,13 +1,17 @@
 package com.mapbox.navigation.ui.maneuver.model
 
+import com.mapbox.api.directions.v5.models.MapboxShield
+
 /**
  * Data structure  that holds information about road shield.
  * @property shieldUrl String
  * @property shieldIcon ByteArray
+ * @property mapboxShield MapboxShield
  */
 data class RoadShield(
     val shieldUrl: String,
-    var shieldIcon: ByteArray
+    val shieldIcon: ByteArray,
+    val mapboxShield: MapboxShield? = null
```

Comment:
You should add `@JvmOverloads` annotation to the primary constructor.

Reply:
Fixed.